src/components/StatsBoxGrid.tsx

```diff
@@ -233,18 +127,90 @@ const GridItem: React.FC<IGridItemProps> = ({ metric, dir }) => {
   )
 
   return (
-    <Box>
+    <Box
+      position="relative"
+      color="text"
+      height="20rem"
+      background={""}
```

Comment:
I believe this is not needed as there is no default `background` color coming from anywhere else.
```suggestion
```